passport-config.js

```diff
@@ -5,23 +5,23 @@ const bcrypt = require('bcrypt')
 function initialize(passport) {
     const authenticateUser = async (email, password, done) => {
         Users.findOne({ $or: [{ 'email': email.toLowerCase() }, { 'name': email.toLowerCase() }] }, async (err, user) => {
-            if (!user) return done(null, false, { message: 'No user with that email' })
+            if (!user) return done(null, false, { message: 'No users found using that email!' })
```

Comment:
user is singular because we're referring to a singular user

Reply:
users/using sounds clucky doesnt come off the tongue easily